btr-web/btr-layouts/components/bcros/BusinessDetails.vue

```diff
@@ -51,7 +51,7 @@ function loadComponentData (identifier: string) {
 }
 // watcher required because layouts start rendering before the route is initialized
 watch(() => route.params.identifier as string, loadComponentData)
-onMounted(() => {
+onBeforeMount(() => {
   // onMounted required for refresh case (route will be set already so ^ watcher will not fire)
```

Comment:
comment needs updating, is it not required anymore ?

Reply:
will update the comment -- the before mount just ensures this happens before the user sees the page rather than showing 'Not Available' for half a second